doc/sphinx/Pacemaker_Explained/alerts.rst

```diff
@@ -1,422 +1,496 @@
+
+   .. code-block:: xml
+
+      <configuration>
+         <alerts>
+            <alert id="my-alert" path="/path/to/my-script.sh">
                <select>
-                 <select_nodes />
-                 <select_fencing />
+                  <select_nodes />
+                  <select_fencing />
                </select>
-               <recipient id="my-alert-recipient1" value="someuser@example.com"/>
-           </alert>
-       </alerts>
-   </configuration>
-   -----
-   =====
-   
-   The possible options within +<select>+ are +<select_nodes>+,
-   +<select_fencing>+, +<select_resources>+, and +<select_attributes>+.
-   
-   With +<select_attributes>+ (the only event type not enabled by default), the
-   agent will receive alerts when a node attribute changes. If you wish the agent
-   to be called only when certain attributes change, you can configure that as well.
-   
-   .Alert configuration to be called when certain node attributes change
-   =====
-   [source,XML]
-   -----
-   <configuration>
-       <alerts>
-           <alert id="my-alert" path="/path/to/my-script.sh">
+               <recipient id="my-alert-recipient1"
+                          value="someuser@example.com"/>
+            </alert>
+         </alerts>
+      </configuration>
+   
+The possible options within ``<select>`` are ``<select_nodes>``,
+``<select_fencing>``, ``<select_resources>``, and ``<select_attributes>``.
+
+With ``<select_attributes>`` (the only event type not enabled by default), the
+agent will receive alerts when a node attribute changes. If you wish the agent
+to be called only when certain attributes change, you can configure that as well.
+   
+.. topic:: Alert configuration to be called when certain node attributes change
+
+   .. code-block:: xml
+
+      <configuration>
+         <alerts>
+            <alert id="my-alert" path="/path/to/my-script.sh">
                <select>
-                 <select_attributes>
-                   <attribute id="alert-standby" name="standby" />
-                   <attribute id="alert-shutdown" name="shutdown" />
-                 </select_attributes>
+                  <select_attributes>
+                     <attribute id="alert-standby" name="standby" />
+                     <attribute id="alert-shutdown" name="shutdown" />
+                  </select_attributes>
                </select>
                <recipient id="my-alert-recipient1" value="someuser@example.com"/>
-           </alert>
-       </alerts>
-   </configuration>
-   -----
-   =====
-   
-   Node attribute alerts are currently considered experimental. Alerts may be
-   limited to attributes set via attrd_updater, and agents may be called multiple
-   times with the same attribute value.
+            </alert>
+         </alerts>
+      </configuration>
    
+Node attribute alerts are currently considered experimental. Alerts may be
+limited to attributes set via attrd_updater, and agents may be called multiple
```

Comment:
attrd_updated seems like text that should be formatted like code or a command line.  It wasn't previously, but it sure seems like it should be.

Reply:
good point, done